pkg/osquery/table/launcher_info.go

```diff
@@ -60,13 +62,23 @@ func generateLauncherInfoTable(store types.GetterSetter) table.GenerateFunc {
 			return nil, err
 		}
 
-		publicKey, fingerprint, err := osquery.PublicRSAKeyFromDB(store)
+		publicKey, fingerprint, err := osquery.PublicRSAKeyFromDB(configStore)
 		if err != nil {
 			// No logger here, so we can't easily log. Move on with blank values
 			publicKey = ""
 			fingerprint = ""
 		}
 
+		uptimeBytes, _ := upTimeHistoryStore.Get([]byte("process_start_time"))
+		uptime := ""
+
+		if uptimeBytes != nil {
+			startTime, _ := time.Parse(time.RFC3339, string(uptimeBytes))
```

Comment:
should probably check the error here before processing `startTime`

Reply:
Done